changelog.json

```
@@ -1,5 +1,26 @@
 {
   "releases": {
+    "2.9.1": [
+      "[Added] Add Fluent Terminal shell support - #12305. Thanks @Idered!",
+      "[Added] Add support for IntelliJ CE for macOS - #12748. Thanks @T41US!",
+      "[Added] Show number of lines changed in a commit - #11656",
+      "[Fixed] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
```

I think this should also be moved to the improved section.
```suggestion
      "[Improved] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
```